src/app/globals.css

```diff
@@ -20,6 +20,20 @@ body {
   font-family: Arial, Helvetica, sans-serif;
 }
 
+.dark {
+  --Dark-Blue: hsl(209, 23%, 22%);
+  --Very-Dark-Blue: hsl(207, 26%, 17%);
+  --Very-Dark-Blue: hsl(200, 15%, 8%);
+  --Dark-Gray: hsl(0, 0%, 52%);
```

Comment:
Hi @hassandogo. It's not good to have "Very Dark Blue" twice. Please rename accordingly